src/defu.ts

```diff
@@ -39,9 +40,17 @@ function _defu<T>(
         value,
         object[key],
         (namespace ? `${namespace}.` : "") + key.toString(),
-        merger
+        merger,
+        clone
       );
     } else {
+      if (clone) {
+        // eslint-disable-next-line no-var
+        var structuredClone: (x: any) => any;
+        value = structuredClone
+          ? structuredClone(value)
+          : JSON.parse(JSON.stringify(value));
```

Comment:
this would remove unserializable objects like functions, won't it?

my object may have an array of functions!